tests/decorators/test_dapr.py

```diff
@@ -0,0 +1,246 @@
+
+        self.assertEqual(len(func.get_bindings()), 2)
+
+        output = func.get_bindings()[0]
+
+        self.assertEqual(output.get_dict_repr(), {
+            "direction": BindingDirection.OUT,
+            "type": DAPR_STATE,
+            "name": "out",
+            "stateStore": "dummy_state_store",
+            "key": "dummy_key"
+        })
+
+    def test_dapr_invoke_output_binding(self):
+        app = self.dapr_func_app
+
+        @app.dapr_service_invocation_trigger(arg_name="req",
+                                             method_name="dummy")
+        @app.dapr_invoke_output(arg_name="out",
+                                app_id="dummy_app_id",
+                                method_name="dummy_method_name",
+                                http_verb="dummy_http_verb")
+        def dummy():
+            pass
+
+        func = self._get_user_function(app)
+
+        self.assertEqual(len(func.get_bindings()), 2)
+
+        output = func.get_bindings()[0]
+
+        self.assertEqual(output.get_dict_repr(), {
+            "direction": BindingDirection.OUT,
+            "type": DAPR_INVOKE,
+            "name": "out",
+            "appId": "dummy_app_id",
+            "methodName": "dummy_method_name",
+            "httpVerb": "dummy_http_verb"
+        })
+
+    def test_dapr_publish_output_binding(self):
+        app = self.dapr_func_app
+
+        @app.dapr_service_invocation_trigger(arg_name="req",
+                                             method_name="dummy")
+        @app.dapr_publish_output(arg_name="out",
+                                 pub_sub_name="dummy_pub_sub_name",
+                                 topic="dummy_topic")
+        def dummy():
+            pass
+
+        func = self._get_user_function(app)
+
+        self.assertEqual(len(func.get_bindings()), 2)
+
+        output = func.get_bindings()[0]
+
+        self.assertEqual(output.get_dict_repr(), {
+            "direction": BindingDirection.OUT,
+            "type": DAPR_PUBLISH,
+            "name": "out",
+            "pubSubName": "dummy_pub_sub_name",
+            "topic": "dummy_topic"
+        })
+
+    def test_dapr_binding_output_binding(self):
+        app = self.dapr_func_app
+
+        @app.dapr_service_invocation_trigger(arg_name="req",
+                                             method_name="dummy")
+        @app.dapr_binding_output(arg_name="out",
+                                 binding_name="dummy_binding_name",
+                                 operation="dummy_operation")
+        def dummy():
+            pass
+
+        func = self._get_user_function(app)
+
+        self.assertEqual(len(func.get_bindings()), 2)
+
+        output = func.get_bindings()[0]
+
+        self.assertEqual(output.get_dict_repr(), {
+            "direction": BindingDirection.OUT,
+            "type": DAPR_BINDING,
+            "name": "out",
+            "bindingName": "dummy_binding_name",
+            "operation": "dummy_operation"
+        })
```

Comment:
@ASHIQUEMD - Have you tested this with Blueprints as well?
@gavin-aguiar - you might need to sync with @ASHIQUEMD for Blueprint test and verification

Reply:
We added DaprBlueprint class and validated the dapr bindings and triggers with it.